### PR TITLE
aws/lumberyard game engine

### PR DESCRIPTION
**Pull request by PatMyron**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [x] Content (and my changes are in `index.md`)

---------------------------------------------------------------------

### CODE REVIEW

**Review by jn23-c — APPROVED** [review]
Great

**Review by jn23-c — APPROVED** [review]

File: collections/game-engines/index.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 items:
+ - aws/lumberyard
```
Comment (MikeMcQuaid) [inline collections/game-engines/index.md:3]: Can you move this to the bottom of the list? Thanks!

Comment (PatMyron) [reply collections/game-engines/index.md]: done

**Review by smashwilson — APPROVED** [review]
:+1: Thanks!

**Pull request merged**